src/Handheld_and_HTPC_edition/Handheld_Wiki/Ayaneo_Handhelds.md

```diff
@@ -34,8 +34,11 @@ Read the [**Installing Bazzite on Handheld PCs documentation**](/General/Install
   - Alternatively, since ALC269VB is an audio chipset that has been around for years, it just need its pins to be rerouted with `hdajackretask` (part of fedora's alsa-tools) package. (You basically need to reroute the auto-generated "green headset" and "black headset" devices to "Channel 30 & 31").
   - External audio works over both HDMI and eGPU.
 - **Suspend does not work without a workaround**!
-  - see suspend workaround [here](https://universal-blue.discourse.group/t/ayaneo-handheld-compatibility/2417#p-5599-suspend-workaround-5)
+  - see suspend workaround [here](https://docs.bazzite.gg/Handheld_and_HTPC_edition/Handheld_Wiki/Ayaneo_Handhelds/#suspend-workaround)
```

Comment:
Urls need to point to markdown files, not the docs.bazzite.gg doman, see https://github.com/KyleGospo/docs.bazzite.gg#1-internal-links
```suggestion
  - see suspend workaround [here](#suspend-workaround)
```